ex02.exs

```diff
@@ -94,11 +95,14 @@ defmodule Ex02 do
     # « your code goes here»
     # You can give an agent a name by adding the
     # `name: @global_name` option to start_link
+    { :ok, _ } = Agent.start_link(fn -> initial_value end, name: @global_name)
   end
 
   def global_next_value do
     # « your code goes here»
     # and you can use that name instead of a pid
     # in calls such as `get_and_update`
+    Agent.get_and_update(:my_global_agent, fn state -> {state, state + 1} end)
```

Comment:
Why use `@global_name` to create the agent, but `:my_global_agent` to access it?

# -2